plugins/MantisCoreFormatting/files/markdown.css

```diff
@@ -1,3 +1,24 @@
+/*
+ * Rules taken from bootstrap.
+ *
+ * The right way is creating boostrap css from scss sources instead
+ * of override the rules here.
+ *
+ * @todo fix this directly in bootstrap and delete this rules.
+ */
+code {
+    color: #333;
+    white-space: pre;
```

Comment:
I don't think that's right; why are you adding that ?

Inline quotes are expected to wrap with text and with `pre` there would no line breaks introduced, so the \<code> block would extend the parent \<td> horizontally which is not wanted.

For me it's fine if whitespace is collapsed in this context

Reply:
Github is using `white-space: break-spaces;` in this context

Reply:
> I don't think that's right; why are you adding that ?

I can't give you a proper anwser. I can't remember what I was thinking … probably nothing. Maybe it was all too fast, driven by the desire to get the Markdown theme done quickly.

Thanks for the correction.

> For the record

:-) they do it better.